openapi/anchor-platform/Callbacks API.yml

```diff
@@ -466,3 +466,34 @@ paths:
             application/json:
               schema:
                 $ref: './schemas.yml#/components/schemas/Error'
+  /events:
+    post:
+      tags:
+        - Events
+        - SEP-24
+        - SEP-31
+      summary: Receive a event from the anchor platform.
+      operationId: postEvent
+      description: |
+        Receive a JSON object representing an event.
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                timestamp:
+                    type: string
+                    description: |
+                      `timestamp` the current Unix timestamp (number of seconds since epoch) at the time the callback is sent.
+                payload:
+                  $ref: './schemas.yml#/components/schemas/Event'
+      responses:
+        '200':
+          description: The event is successfully processed and ready to receive next event. The response body is empty.
+        '400':
+          description: The event is invalid or rejected. The response body contains the error message.
```

Comment:
I'm not sure we want to define any non-200 response code. All responses should have 200, and all other response codes will be interpreted as an error and retries will be made according to the AP's strategy.

Reply:
The response code is limited `400`. If the business server returns anything other than 400, they will be considered the same way as `400`. 

The retries can be handled in the next fix version.

Reply:
But 400 itself is not a valid status, so I don't think we should define it in the spec.